datafusion/functions/src/string/common.rs

```diff
@@ -214,6 +214,23 @@ where
                 i64,
                 _,
             >(array, op)?)),
+            DataType::Utf8View => {
+                let string_array = as_string_view_array(array)?;
```

Comment:
Not sure happy with this implementation, but it seems to work. I tried something akin to:

```rust
let result = string_array.iter().map(|string| string.map(op));
```

but that seems to require a clone of `op` which isn't allowed based on the existing function definition, and seems worth avoiding.

Reply:
this implemenation looks reasonable to me. Thanks @tshauck